test/browser/createContext.test.js

```diff
@@ -352,6 +353,10 @@ describe('createContext', () => {
 	});
 
 	it('should not re-render the consumer if the context doesn\'t change', () => {
+
+		// Disable the debounce rendering
+		options.debounceRendering = f => f();
```

Comment:
That should be done via `act()`. That we we don't have to manually restore it :)